<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FluorophoreModelDialog</class>
 <widget class="QDialog" name="FluorophoreModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fluorophore Model Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gui_GroupBox">
     <property name="title">
      <string>Fluorophore Model</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="gui_ColorChannelGroupBox">
        <property name="title">
         <string>Channel</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="gui_ColorChannelLabel">
           <property name="text">
            <string>Color channel:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="gui_ColorChannelComboBox">
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="gui_EnabledCheckBox">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="gui_samplingModeGroupBox">
        <property name="title">
         <string>Sampling Mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="gui_useFixedDensity">
           <property name="text">
            <string>Use Fixed Density</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="gui_useFixedNumberOfFluorophores">
           <property name="text">
            <string>Use Fixed Number of Fluorophores</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gui_AreaLabel">
        <property name="text">
         <string>Surface Area:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="gui_AreaEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="gui_DensityEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gui_DensityLabel">
        <property name="text">
         <string>Density:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="gui_NumberOfFluorophoresEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="gui_NumberOfFluorophoresLabel">
        <property name="text">
         <string>Number of fluorophores:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QSlider" name="gui_NumberOfFluorophoresSlider">
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gui_ClustersGroupBox">
     <property name="title">
      <string>Clusters</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="gui_UseSingleFluorophorePerSampleRadioButton">
        <property name="text">
         <string>Use one fluorophore per sample</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="gui_UseRingClusterRadioButton">
        <property name="text">
         <string>Use ring cluster per sample</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="gui_FluorophoresAroundRingLabel">
        <property name="text">
         <string>Fluorophores around ring:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="gui_FluorophoresAroundRingEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="gui_RingRadiusLabel">
        <property name="text">
         <string>Ring radius:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="gui_RingRadiusEdit"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="gui_RandomizeRingOrientationsCheckBox">
        <property name="text">
         <string>Randomize ring orientations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FluorophoreModelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FluorophoreModelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
